tests/test_ble_lifecycle_receive_targets.py

```
@@ -356,222 +372,276 @@ def test_lifecycle_disconnect_planning_and_side_effect_paths(
-            DummyClient(),
-            "device-key",
-            "alias-key",
-            is_closing=True,
-            lost_gate_ownership=False,
-            restore_address=None,
-            restore_last_connection_request=None,
+        iface._ever_connected = MagicMock()
+        assert BLELifecycleService._ever_connected_flag(iface) is False
+
+        snapshots = [
+            _OwnershipSnapshot(True, False, False, False),
```

Ruff (0.15.6)

[warning] Boolean positional value in function call

(FBT003)